src/ParagraphNodeSpec.js

```diff
@@ -12,12 +12,6 @@ export const INDENT_MARGIN_PT_SIZE = 36;
 export const MIN_INDENT_LEVEL = 0;
 export const MAX_INDENT_LEVEL = 7;
 export const ATTRIBUTE_INDENT = 'data-indent';
-export const LINE_SPACING_VALUES = [
-  '100%',
-  '115%',
-  '150%', // Default value.
```

Comment:
is this going to affect existing docs, since the default is changing from 150% to "single"?